packages/wonder-blocks-accordion/src/components/accordion.tsx

```diff
@@ -120,13 +120,19 @@ const Accordion = React.forwardRef(function Accordion(
         ...ariaProps
     } = props;
 
+    // Starting array for the initial expanded state of each section.
     const startingArray = Array(children.length).fill(false);
+    // If initialExpandedIndex is specified, we want to open that section.
     if (initialExpandedIndex !== undefined) {
         startingArray[initialExpandedIndex] = true;
     }
-
     const [sectionsOpened, setSectionsOpened] = React.useState(startingArray);
 
+    // Setting up focus state and refs for keyboard navigation.
+    const [currentlyFocusedSection, setCurrentlyFocusedSection] =
+        React.useState(0);
+    const childRefs = Array(children.length).fill(null);
```

Comment:
**thought:** One important thing to consider here is to filter out any possible `disabled` accordion sections. Are we including them as part of the focus order?

Reply:
Documenting our discussion from a separate meeting: I think we should keep the disabled sections in the focus order since they'd receive focus anyway as part of aria-disabled. Screenreaders should read them out as disabled anyway, so it should still be clear. I'll add a gray outline to make it more obvious for visual users.

Reply:
Awesome thanks! It would be nice adding a summary of what you said here as a code comment :).